sym-link.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#  This is for setting up your sym links for dev.  It is needed for testing.
+#  You may modify the DIR below for your environment.
+#  enter 1 as first argument for PSIJ 
+PSIJ=$1
+
+DIR=~/psi-j-testing-service/src/psij/web/instance
+
+cd $DIR
+rm customization.js 
+
+if [ $PSIJ -eq 1 ]
+then
+   ln -s customization-psij.js customization.js
+else
+   ln -s customization-sdk.js customization.js
+fi
+
+ls -al $DIR
```

Comment:
I don't think we need a script because this should not be something that needs to be done repeatedly. We should put the information that you could link one of the two on deployment in one of the relevant readme files.